addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/BufferOverflowScanRule.java

```diff
@@ -188,4 +190,18 @@ private String randomCharacterString(int length) {
         }
         return sb1.toString();
     }
+
+    @Override
+    public List<Alert> getExampleAlerts() {
+        List<Alert> alerts = new ArrayList<>();
+        alerts.add(
+                newAlert()
+                        .setConfidence(Alert.CONFIDENCE_MEDIUM)
+                        .setParam("param")
+                        .setAttack(randomCharacterString(2100))
```

Comment:
Hardcode a value, if we ever use the attack in the docs we don't want it to change each time.